Makefile

```diff
@@ -1,7 +1,12 @@
 # pinned versions
-FABRIC_VERSION=2.2
+FABRIC_VERSION ?= 2.2.8
+FABRIC_TWO_DIGIT_VERSION = $(shell echo $(FABRIC_VERSION) | cut -d '.' -f 1,2)
 ORION_VERSION=v0.2.5
 
+# need to install fabric binaries outside of fsc tree for now (due to chaincode packaging issues)
+FABRIC_BINARY_BASE=$(PWD)/../fabric
+FAB_BINS=$(FABRIC_BINARY_BASE)/bin
```

Comment:
If I want to use my custom bins within this makefile, should we be able to override like this ... `FAB_BINS ?= $(FABRIC_BINARY_BASE)/bin` ??

Reply:
Yep, that's a great idea, thanks